colibre/scripts/stellar_abundances.py

```diff
@@ -14,6 +14,43 @@
 import h5py
 
 
+def make_hist(x, y, cut, xi, yi):
+
+    selection = np.where(cut)[0]
+
+    # Create a histogram
+    h, xedges, yedges = np.histogram2d(
+        x[selection], y[selection], bins=(xi, yi), normed=True
+    )
+
+    return h, xedges, yedges
+
+
+def make_stellar_abundance_distribution(x, y, R, z, xi, yi):
+
+    distance_cut = (R >= 0) & (R < 3) & (np.abs(z) >= 1) & (np.abs(z) < 2)
```

Comment:
- Please add a comment explaining the units of the numbers

- I'd also suggest using `np.logical_and()` instead of `&`. You could create `mask_z = ...` and `mask_R  = ..` and then combine them by `mask_total = np.logical_and(...)`

- Note also that you could shorten the code below immensely by writing something like

```
h = array with zeros in the correct shape
for mask in masks:
    h += make_hist(x, y, mask, xi, yi)[0]
```

Reply:
done!